eval/src/vespa/eval/tensor/sparse/sparse_tensor_match.cpp

```diff
@@ -73,9 +72,9 @@ transformAddress(SparseTensorAddressBuilder &builder,
 
 
 void
-SparseTensorMatch::fastMatch(const TensorImplType &lhs,
-                                const TensorImplType &rhs)
+SparseTensorMatch::fastMatch(const TensorImplType &lhs, const TensorImplType &rhs)
 {
+    _builder.reserve(lhs.cells().size());
```

Comment:
since the function is called 'reserve', I think the * 2 should be outside (here) and not inside the function